### ISSUE TITLE
missing bonds

### ISSUE THREAD

**Issue by anhi** [BALL-Project/ball#39]
**Reported by Julia Fischer on 1 Dec 38774446 11:06 UTC**
I installed BALLView with the Ubuntu installer on a Kubuntu-Linux System. I work on an amd64 machine
but 32bit Linux is installed. BALLView gets installed correctly but when I open any PDB file
BALLView can not display any kind of bond representation. SES, SAS and VDW work fine, BALL&STICK,
STICK and LINE show only the atoms and not the bonds, and finally BACKBONE and CARTOON don't show
anything. When starting BALLView on the console , it pints things like:
reading PDB file...
Read 1011 atoms from file "/home/julia/Desktop/benchmark/ET/1P0Z_A.pdb"
> applying molecular properties ...
  > normalized names
 > generate missing bonds - failed:
InvalidFormat @ string.C:505: problem converting '1.33' to a number.
I uninstalled the package and tried compiling BALL totally but with the same result. BALLView worked
on this machine before with Gentoo-Linux, so I guess it must be a software problem. It is
definitively not because of conrupt files. I tried downloading them with BALLView or directly from
PDB and I tried many different files. Moreover I have generated nice pictures of the exact file in
the error message from my old installation, where cartoon and backbone worked fine.

**Comment by anhi** [comment]
**Commented by admin on 13 Nov 38774447 16:26 UTC**
This is problem generated by the locale settings on your computer. The
conversion of floating point numbers fails if you have set the language to
german. Until we come up with a better solution, you may try
export LANG=C
before starting BALLView or any other BALL application. This should fix your
problems...

**Comment by anhi** [comment]
**Commented by admin on 25 Feb 38774448 20:26 UTC**
This should be fixed in recent distributions... If this bug reappears, please
add a new issue to the bug tracker.
